app/src/main/AndroidManifest.xml

```diff
@@ -3,6 +3,10 @@
     package="com.nervousfish.nervousfish"
     xmlns:tools="http://schemas.android.com/tools">
 
+    <uses-permission android:name="android.permission.BLUETOOTH" />
+    <uses-permission android:name="android.permission.BLUETOOTH_ADMIN" />
+    <uses-permission android:name="android.permission.ACCESS_COARSE_LOCATION"/>
```

Comment:
That was on stackoverflow, without it bluetooth wasn't discovering other devices, @kcallebaut knows moer about this